jerry-core/parser/js/js-parser.c

```diff
@@ -1726,12 +1715,41 @@ parser_post_processing (parser_context_t *context_p) /**< context */
     *(--base_p) = ECMA_VALUE_EMPTY;
   }
 
+  if (context_p->argument_length != UINT16_MAX)
+  {
+    compiled_code_p->status_flags |= CBC_CODE_FLAGS_HAS_EXTENDED_INFO;
+    *(--base_p) = context_p->argument_length;
```

Comment:
If I see correctly the extended info is now an `ecma_value_t`. How about creating a struct like 

```c
typedef struct {
 union {
    uint8_t argument_length;
    ecma_value_t alignment;
  } u;
  
} ecma_compiled_code_extended_info_t;
```

If any extra info should be added later, extending this structure with fields would be more comfortable.

Reply:
I also consider it as a flagset in the future. I don't know how many bits will be needed for different things, I would let that later.

Reply:
Sure, just we can keep that in mind.